smarts/env/hiway_env.py

```diff
@@ -129,21 +129,61 @@ def __init__(
         if visdom:
             visdom_client = VisdomClient()
 
-        self._smarts = SMARTS(
-            agent_interfaces=agent_interfaces,
-            traffic_sim=SumoTrafficSimulation(
-                headless=sumo_headless,
-                time_resolution=fixed_timestep_sec,
-                num_external_sumo_clients=num_external_sumo_clients,
-                sumo_port=sumo_port,
-                auto_start=sumo_auto_start,
-                endless_traffic=endless_traffic,
-            ),
-            envision=envision_client,
-            visdom=visdom_client,
-            fixed_timestep_sec=fixed_timestep_sec,
-            zoo_addrs=zoo_addrs,
-        )
+        is_opendrive = HiWayEnv.check_scenario_versions(scenarios)
+        if is_opendrive:
+            # We currently don't support the Native SUMO Traffic Provider or Social Agents for OpenDRIVE maps
+            self._smarts = SMARTS(
+                agent_interfaces=agent_interfaces,
+                traffic_sim=None,
+                envision=envision_client,
+                visdom=visdom_client,
+                fixed_timestep_sec=fixed_timestep_sec,
+            )
+
+        else:
+            from smarts.core.sumo_traffic_simulation import SumoTrafficSimulation
+
+            self._smarts = SMARTS(
+                agent_interfaces=agent_interfaces,
+                traffic_sim=SumoTrafficSimulation(
+                    headless=sumo_headless,
+                    time_resolution=fixed_timestep_sec,
+                    num_external_sumo_clients=num_external_sumo_clients,
+                    sumo_port=sumo_port,
+                    auto_start=sumo_auto_start,
+                    endless_traffic=endless_traffic,
+                ),
+                envision=envision_client,
+                visdom=visdom_client,
+                fixed_timestep_sec=fixed_timestep_sec,
+                zoo_addrs=zoo_addrs,
+            )
```

Comment:
I think this also should work:
```suggestion
        traffic_sim = None
        if is_opendrive:
            # We currently don't support the Native SUMO Traffic Provider or Social Agents for OpenDRIVE maps
            pass
        else:
            from smarts.core.sumo_traffic_simulation import SumoTrafficSimulation
            traffic_sim = SumoTrafficSimulation(
                headless=sumo_headless,
                time_resolution=fixed_timestep_sec,
                num_external_sumo_clients=num_external_sumo_clients,
                sumo_port=sumo_port,
                auto_start=sumo_auto_start,
                endless_traffic=endless_traffic,
            )

        self._smarts = SMARTS(
            agent_interfaces=agent_interfaces,
            traffic_sim=traffic_sim,
            envision=envision_client,
            visdom=visdom_client,
            fixed_timestep_sec=fixed_timestep_sec,
            zoo_addrs=zoo_addrs,
        )
```